packages/client/src/__tests__/integration/happy/not-so-exhaustive-schema-mongo/__snapshots__/binary.test.ts.snap

```diff
@@ -559,6 +559,11 @@ export class PrismaClient<
    */
   $runCommandRaw(command: Prisma.InputJsonObject): Prisma.PrismaPromise<Prisma.JsonObject>
 
+
+  /**
+   * Tries to apply pending migrations to the database.
+   */
+  $applyPendingMigrations(): Prisma.PrismaPromise<void>
```

Comment:
Rather than a new method in a public API, could client just automatically do it on startup if it is built for React Native?

Reply:
We talked about this with Soren, and unlike the server version, this process needs to be throwabl, catchable and automatable in case a migration fails since you cannot revert or manually change the databases on the user phones once something has been deployed.

By having it separated it makes this process more explicit and less prone to errors.

Reply:
Ok, then let's make sure those types won't be generated for non-RN clients